src/current/v23.2/read-committed.md

```diff
@@ -0,0 +1,928 @@
+
+- You are [migrating an application to CockroachDB]({% link {{ page.version.version }}/migration-overview.md %}) that was built at a `READ COMMITTED` isolation level on the source database, and it is not feasible to modify your application to use `SERIALIZABLE` isolation.
+
+Whereas `SERIALIZABLE` isolation guarantees data correctness by placing transactions into a [serializable ordering]({% link {{ page.version.version }}/demo-serializable.md %}), `READ COMMITTED` isolation permits some [concurrency anomalies](#concurrency-anomalies) in exchange for minimizing transaction aborts, [retries]({% link {{ page.version.version }}/developer-basics.md %}#transaction-retries), and blocking. Compared to `SERIALIZABLE` transactions, `READ COMMITTED` transactions return fewer [serialization errors]({% link {{ page.version.version }}/transaction-retry-error-reference.md %}) that require client-side handling. See [`READ COMMITTED` transaction behavior](#read-committed-transaction-behavior).
+
+{{site.data.alerts.callout_info}}
+`READ COMMITTED` on CockroachDB provides stronger isolation than `READ COMMITTED` on PostgreSQL. On CockroachDB, `READ COMMITTED` prevents anomalies within single statements. For complete details on how `READ COMMITTED` is implemented on CockroachDB, see the [Read Committed RFC](https://github.com/cockroachdb/cockroach/blob/master/docs/RFCS/20230122_read_committed_isolation.md).
+{{site.data.alerts.end}}
+
+## Enable `READ COMMITTED` isolation
+
+To make `READ COMMITTED` isolation available to use on a cluster, enable the following [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) in the SQL shell:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SET CLUSTER SETTING sql.txn.read_committed_isolation.enabled = 'true';
+~~~
+
+After you enable the cluster setting, you can configure `READ COMMITTED` isolation at the [session](#set-the-current-session-to-read-committed) or [transaction](#set-the-current-transaction-to-read-committed) level.
+
+{{site.data.alerts.callout_info}}
+If the cluster setting is not enabled, `READ COMMITTED` transactions will run as `SERIALIZABLE`.
+{{site.data.alerts.end}}
+
+### Set the current session to `READ COMMITTED`
+
+To set all future transactions in a session to run at `READ COMMITTED` isolation, use one of the following options:
+
+- The [`SET SESSION CHARACTERISTICS`]({% link {{ page.version.version }}/set-vars.md %}#special-syntax-cases) statement:
+
+	{% include_cached copy-clipboard.html %}
+	~~~ sql
+	SET SESSION CHARACTERISTICS AS TRANSACTION ISOLATION LEVEL READ COMMITTED;
+	~~~
+
+- The [`default_transaction_isolation`]({% link {{ page.version.version }}/session-variables.md %}#default-transaction-isolation) session variable:
+
+	At the database level:
+
+	{% include_cached copy-clipboard.html %}
+	~~~ sql
+	SET default_transaction_isolation = 'read committed';
```

Comment:
This would set the default at the cluster level. At the database level would be:
```sql
ALTER DATABASE db SET default_transaction_isolation = 'read committed';
```

Reply:
It would be great to include both.